src/material-experimental/mdc-radio/radio.ts

```diff
@@ -151,11 +159,15 @@ export class MatRadioButton extends _MatRadioButtonBase implements AfterViewInit
     this._radioFoundation.destroy();
   }
 
-  private _setClass(cssClass: string, active: boolean) {
+  setClass(cssClass: string, active: boolean) {
     this._classes = {...this._classes, [cssClass]: active};
     this._changeDetector.markForCheck();
   }
 
+  getChangeDetector() {
```

Comment:
This should also be with an underscore. You could also expose `_changeDetector` and mark it as readonly.